packages/oui-button-guide/button-guide.less

```diff
@@ -0,0 +1,88 @@
+@import "./_variables";
+
+.oui-guide {
+  .oui-action-menu__item {
+    .oui-button_action-menu {
+      &:hover {
+        .oui-icon-external_link {
+          visibility: visible;
+        }
+      }
+      .oui-icon-external_link {
+        visibility: hidden;
+      }
+    }
+  }
+  .oui-guide__footer {
+    .oui-action-menu__item {
+      .oui-button_action-menu {
+        text-align: center;
+        font-weight: @oui-font-semibold;
+        text-transform: capitalize;
+      }
+    }
+    .oui-button_secondary {
+      text-align: center;
+      padding-top: @oui-guide-footer-button-top-padding;
+      width: @oui-guide-footer-button-width;
+      margin: @oui-guide-footer-button-margin;
+    }
+  }
+
+  .oui-guide__body {
+    max-height: 70vh;
+    overflow-y: auto;
+  }
+
+  //override oui-dropdown size
+  .oui-dropdown__content {
+    width: @oui-guide-dropdown-content-width;
+
+    &_bottom {
```

Comment:
should be &-bottom. It's not a modifier.

Reply:
This is the class in oui-dropdown. overriding this